OneBusAway/ui/trip_details/VehicleMapController.swift

```diff
@@ -260,4 +253,23 @@ class VehicleMapController: UIViewController, MKMapViewDelegate {
 
         return annotationView
     }
+    
+    // MARK: - UI Configurations
+    
+    func createVehicleMapHoverBar() {
+        let VehicleMapHoverBar = ISHHoverBar()
+        self.view.addSubview(VehicleMapHoverBar)
+        
+        let toggleButton = UIBarButtonItem()
+        self.toggleButton.addTarget(self, action: #selector(toggleButtonTapped), for: .touchUpInside)
+        toggleButton.customView = self.toggleButton
+
+        let recenterMapButton = UIBarButtonItem(image: UIImage(named: "Map_Selected"), landscapeImagePhone: nil, style: .plain, target: self, action: #selector(self.recenterMap))
+        
+        VehicleMapHoverBar.items = [recenterMapButton, toggleButton]
+        VehicleMapHoverBar.snp.makeConstraints { (make) in
+            make.trailing.equalTo(mapView).inset(UIEdgeInsetsMake(0, 0, 0, 8))
```

Comment:
SnapKit allows for very concise definitions of constraints. This can be consolidated into:

```
make.trailing.bottom.equalToSuperview().inset(UIEdgeInsetsMake(0, 0, 8, 8))
```